Code/GraphMol/Depictor/RDDepictor.cpp

```diff
@@ -125,7 +121,7 @@ std::list<EmbeddedFrag>::iterator _findLargestFrag(
     std::list<EmbeddedFrag> &efrags) {
   std::list<EmbeddedFrag>::iterator mfri;
   int msiz = 0;
-  for (auto efri = efrags.begin(); efri != efrags.end(); efri++) {
+  for (auto efri = efrags.begin(); efri != efrags.end(); ++efri) {
```

Comment:
Possibly use range

Reply:
same story with using the iterator elsewhere